test/command.js

```diff
@@ -330,6 +330,11 @@ test('$ stdin defaults to "inherit"', async t => {
 	t.is(stdout, 'foo');
 });
 
+test('$ stdin also defaults to "pipe"', async t => {
+	const {stdout} = await $`noop.js foo`.pipeStdout($`stdin.js`);
```

Comment:
Note: `.pipeStdout()` will need to be renamed to `.pipe()` once https://github.com/sindresorhus/execa/pull/757 is merged.